docs/resources/chain-integration.md

```diff
@@ -1,42 +1,53 @@
-# Crypto.org Chain Integration documentation
+# Cronos Integration documentation 
 
 ## Useful Links
 
-- [Crypto.org Chain website](https://crypto.org/)
-- [GitHub Repository](https://github.com/crypto-org-chain/chain-main)
-- [Official Documentation](https://crypto.org/docs/)
+- [Cronos website](https://cronos.crypto.org/)
+- [GitHub Repository](https://github.com/crypto-org-chain/cronos)
+- [Official Documentation](https://cronos.crypto.org/docs/)
+- [Faucet](https://cronos.crypto.org/faucet/) (Note that the faucet only takes eth-type `0x...`  addr)
+- [Binaries](https://github.com/crypto-org-chain/cronos/releases)
+- [Seeds for fullnode](https://github.com/crypto-org-chain/cronos-testnets/blob/main/testnet.json#L2)
+- [Testnet genesis](https://github.com/crypto-org-chain/cronos-testnets/blob/main/cronostestnet-338/genesis.json)
 
-## Node and RPC setup notes
-
-[Node Setup and RPC note](./node-and-rpc-setup-notes.md)
 
 ## Setup Guide
 
+- **Cronos Testnet**:
+    - [Joining the Cronos Testnet](https://cronos.crypto.org/docs/getting-started/)
+    - [Using MetaMask](https://cronos.crypto.org/docs/getting-started/metamask.html)
+    - [Deploying Smart Contract](https://cronos.crypto.org/docs/getting-started/cronos-smart-contract.html)
+
 - **Mainnet**:
     - [Running a full node;](https://crypto.org/docs/getting-started/mainnet.html)
     - [Running a validator;](https://crypto.org/docs/getting-started/mainnet_validator.html)
     - [Mainnet Validator Security Checklist.](https://crypto.org/docs/getting-started/security-checklist.html#part-1-conduct-survey-on-general-controls-of-hosting-data-centre)    
```

Comment:
May also remove this as people could confused that there is already a Cronos mainnet?